addon/webextension/background/senderror.js

```diff
@@ -1,38 +1,38 @@
-/* globals chrome, communication, makeUuid, Raven, catcher, auth */
+/* globals browser, chrome, communication, makeUuid, Raven, catcher, auth */
 
 window.errorpopup = (function () {
   let exports = {};
 
   let messages = {
     REQUEST_ERROR: {
-      title: "Page Shot is out of order.",
-      info: "Your shot was not saved.  We apologize for the inconvenience. Try again soon."
+      title: browser.i18n.getMessage("requestError"),
+      info: browser.i18n.getMessage("requestErrorDetails")
     },
     CONNECTION_ERROR: {
-      title: "Cannot connect to the Page Shot server.",
-      info: "There may be a problem with the service or with your network connection."
+      title: browser.i18n.getMessage("connectionError"),
+      info: browser.i18n.getMessage("connectionErrorDetails")
     },
     LOGIN_ERROR: {
-      title: "Page Shot is out of order.",
-      info: "Your shot was not saved.  There was an error authenticating with the server."
+      title: browser.i18n.getMessage("requestError"),
+      info: browser.i18n.getMessage("loginErrorDetails")
     },
     LOGIN_CONNECTION_ERROR: {
-      title: "Cannot connect to the Page Shot server.",
-      info: "There may be a problem with the service or your network connection."
+      title: browser.i18n.getMessage("connectionError"),
+      info: browser.i18n.getMessage("loginConnectionErrorDetails")
     },
     UNSHOOTABLE_PAGE: {
-      title: "Page cannot be screenshotted",
-      info: "This is not a normal web page, and Page Shot cannot capture screenshots from it."
+      title: browser.i18n.getMessage("unshootablePageError"),
+      info: browser.i18n.getMessage("unshootablePageErrorDetails")
     },
     SHOT_PAGE: {
-      title: "You can't take a shot of a Page Shot page!"
+      title: browser.i18n.getMessage("recursivePageShotShotError")
```

Comment:
How come SHOT_PAGE and MY_SHOTS don't have a info, and use a really title instead?